master/reference/cni-plugin/configuration.md

```diff
@@ -83,6 +83,26 @@ Optionally, the list of possible IPv4 and IPv6 pools can also be specified via t
 * `ipv4_pools`: An array of CIDR strings (e.g. `"ipv4_pools": ["10.0.0.0/24", "20.0.0.0/16"]`)
 * `ipv6_pools`: An array of CIDR strings (e.g. `"ipv6_pools": ["2001:db8::1/120"]`)
 
+Example CNI config:
+
+```json
+{
+    "name": "any_name",
+    "type": "calico",
+    "policy": {
```

Comment:
I'd drop the `policy` section here since we're not in the k8s section of the doc, and we're focusing on IPAM.